cmd/snap/cmd_find.go

```diff
@@ -160,6 +161,7 @@ func init() {
 		return &cmdFind{}
 	}, map[string]string{
 		"private": i18n.G("Search private snaps"),
+		"narrow":  i18n.G("Only search for snaps in 'stable' and for the current architecture"),
```

Comment:
are we sure wide find other archs, it doesn't seem to be the case. I expect some potential bikeshedding on the name of this option

Reply:
I'm sure we don't, now. I'll fix.